Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CartesianYeeAlgorithm.H

```diff
@@ -223,6 +226,131 @@ struct CartesianYeeAlgorithm {
 
 #endif
 
+#ifdef WARPX_MAG_LLG
```

Comment:
Since this is specific to LLG and mainly does laplacian, lets move this to a different file, in the same directory
`CartesianYeeLaplacian.H`
This would avoid conflicts when we merge and keep things clean